ads/google/a4a/utils.js

```diff
@@ -168,7 +168,6 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
-        {name: 'ea', value: '0'},
```

Comment:
So we removed it from Delayed recently?  The param sweep we did back in early Jan still had it heavily represented in Delayed: https://docs.google.com/spreadsheets/d/1j39sSfFmtGc_y5EVyyHiYMzx8a-vH_ISoU0tLPQTlik/edit#gid=0